main.py

```diff
@@ -59,16 +56,10 @@
     #     EXPERIMENT_ID='Experiment2_INFERENCE_ONLY',
     #     ONLY_INFERENCE=True,
     #     EPOCHS=5,
-    #     ROLE='Both',
+    #     ROLE='Car1',
```

Comment:
Define role in Config and use it here

Reply:
already in experiment :     # Model and Training Configuration
    ROLE: str = ""  # Which car is using the DRL model. Car1, Car2,Both

Reply:
Done